Contest_API.md

```diff
@@ -1296,7 +1296,7 @@ Properties of account objects:
 | id                | ID        | Identifier of the account.
 | username          | string    | The account username.
 | password          | string ?  | The account password.
-| type              | string ?  | The type of account, e.g. `team`, `judge`, `admin`, `analyst`, `staff`.
+| type              | string    | The type of account, e.g. `team`, `judge`, `admin`, `analyst`, `staff`.
```

Comment:
Would there be any accounts that don't fall under one of these types (or even another not listed here)? If not, then I think this should be non-nullable.

Reply:
Since it's just a string and not an "enum" (compare with people:sex) it would certainly fall under *some* possible value. If nothing else `other` would do.

Reply:
Yeah I think `other` is fine? For us for example we have accounts that we otherwise should not expose (like a `CDS`or `kattis` user). Or should we really not expose them? 🤔